src/test/groovy/org/sharegov/mdcgis/PropertyInfoServiceTests.groovy

```diff
@@ -124,12 +109,21 @@ class PropertyInfoServiceTests {
 
 	@Test
 	public void testGetCondoPropertyInfo_NoData(){
-		Map data = [:]
-		assert propertyInfoService.getCondoPropertyInfo(data) == null
-		data = null
-		assert propertyInfoService.getCondoPropertyInfo(data) == null
+		try {
+			Map data = [:]
+			def result = propertyInfoService.getCondoPropertyInfo(data)
+			assert false
+
```

Comment:
not sure if  I understand this. assert false but then you seem to have more code afterwards